pymc/step_methods/metropolis.py

```diff
@@ -1056,7 +1036,7 @@ def delta_logp(
     logp: at.TensorVariable,
     vars: List[at.TensorVariable],
     shared: Dict[at.TensorVariable, at.sharedvar.TensorSharedVariable],
-) -> aesara.compile.Function:
+):
```

Comment:
Wasn't this type hint correct?

Reply:
yes, iut was correct. Now it's taken from one level deeper: `compile_pymc` had the wrong return type hint